### PR TITLE
Add a paddle version of TD3

### PR DESCRIPTION
**Pull request by ShuaibinLi**
### Add files:
mujoco_agent.py, mujoco_model.py, train.py, README.md, ---> benchmark/paddle/TD3/
td3.py ---> parl/algorithms/paddle/

### Change file:
parl/algorithms/paddle/init.py

### CODE REVIEW

File: parl/algorithms/paddle/td3.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        self.policy_noise = policy_noise
+        self.noise_clip = noise_clip
+        self.policy_freq = policy_freq
+        self.total_it = 0
+
+        self.model = model
+        self.target_model = deepcopy(self.model)
+        self.actor_optimizer = paddle.optimizer.Adam(
+            learning_rate=actor_lr,
+            parameters=self.model.actor_model.parameters())
```
Comment (rical730) [inline parl/algorithms/paddle/td3.py]: self.model.get_actor_params()

File: parl/algorithms/paddle/td3.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        self.critic_optimizer = paddle.optimizer.Adam(
+            learning_rate=critic_lr,
+            parameters=self.model.critic_model.parameters())
```
Comment (rical730) [inline parl/algorithms/paddle/td3.py]: self.model.get_critic_params()

File: benchmark/paddle/TD3/mujoco_model.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        self.critic_model = Critic(obs_dim, action_dim)
+
+    def policy(self, obs):
+        return self.actor_model(obs)
+
+    def value(self, obs, action):
+        return self.critic_model(obs, action)
+
+    def Q1(self, obs, action):
+        return self.critic_model.Q1(obs, action)
```
Comment (rical730) [inline benchmark/paddle/TD3/mujoco_model.py:34]:
```
    def get_actor_params(self):
        return self.actor_model.parameters()

    def get_critic_params(self):
        return self.critic_model.parameters()
```

File: benchmark/paddle/TD3/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+## Reproduce TD3 with PARL
+Based on PARL, the TD3 algorithm of deep reinforcement learning has been reproduced, reaching the same level of indicators as the paper in Mujoco benchmarks.
+
+Include following approaches:
```
Comment (TomorrowIsAnOtherDay) [inline benchmark/paddle/TD3/README.md]: Approaches -> improvements:

File: benchmark/paddle/TD3/README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
++ Clipped Double Q-learning
++ Target Networks and Delayed Policy Update
++ Target Policy Smoothing Regularization
+
+> TD3 in
+[Addressing Function Approximation Error in Actor-Critic Methods](https://arxiv.org/abs/1802.09477)
+
+### Mujoco games introduction
+Please see [here](https://github.com/openai/mujoco-py) to know more about Mujoco games.
```
Comment (TomorrowIsAnOtherDay) [inline benchmark/paddle/TD3/README.md]: see -> view

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
LGTM.

**Pull request merged**